op-service/predeploys/addresses.go

```diff
@@ -136,12 +140,20 @@ func init() {
 		Address:       Permit2Addr,
 		ProxyDisabled: true,
 	}
-	Predeploys["SenderCreator"] = &Predeploy{
-		Address:       SenderCreatorAddr,
+	Predeploys["SenderCreator_v060"] = &Predeploy{
+		Address:       SenderCreator_v060Addr,
 		ProxyDisabled: true,
 	}
-	Predeploys["EntryPoint"] = &Predeploy{
```

Comment:
Want to double check that changing this name isn't backwards incompatible somewhere. Looked around the codebase and didn't find a usage, so want to check with the team.

Reply:
Yes this is fine, I don't believe this is used anywhere anymore, previously it was used for L2 genesis generation but now that we do that in solidity, this is left over